### PR TITLE
Create a .editorconfig with 2 spaces indent for js

### PR DESCRIPTION
**Pull request by MetalMatze**

### CODE REVIEW

**Comment by MetalMatze** [comment]
I've added HTML, CSS & Go.

The HTML & CSS styles are based on http://codeguide.co/ which I try to incorporate as much as I can.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+insert_final_newline = true
+
+[*.{html,css,js}]
+charset = utf-8
+indent_style = space
+indent_size = 2
+
+[*.go]
+indent_style = tab
+indent_size = 8
```
Comment (tboerger) [inline .editorconfig]: I would prefer 4 :)

Comment (MetalMatze) [inline .editorconfig]: For `{html,css,js}`? If so, I really don't care. I just like that there's codeguide.co which one can point to if some code isn't properly formatted. Beyond just indentations.

Comment (tboerger) [inline .editorconfig]: 2 for HTML is fine for me. I just prefer 4 for go

Comment (mikkeloscar) [inline .editorconfig]: It doesn't really make sense to force an indent size when using tab.

Comment (tboerger) [inline .editorconfig]: It does, but only visual within the editor

Comment (mikkeloscar) [inline .editorconfig]: That is what i meant. It doesn't affect the code if you use 4 in your editor and I use 8 in mine, thus it shouldn't be forced in an `.editorconfig`.

**Pull request closed without merging**